File: dart/pangram/lib/pangram.dart
class Pangram {
  static String alpha = 'abcdefghijklmnopqrstuvwxyz';
  bool isPangram(String word) {
    word = word.toLowerCase();
    for (int i = 0; i < alpha.length; i++) {
      if (word.indexOf(alpha[i]) == -1) {
        return false;
      }
    }
    return true;
  }
}

File: csharp/pangram/Pangram.cs
using System;

public static class Pangram
{
    private static string alpha = "abcdefghijklmnopqrstuvwxyz";
    public static bool IsPangram(string input)
    {
        string lower = input.ToLower();
        foreach (char c in alpha) {
            if (lower.IndexOf(c) == -1) {
                return false;
            }
        }
        return true;
    }
}
